homeassistant/components/remote/services.yaml

```diff
@@ -58,9 +58,25 @@ learn_command:
     command:
       description: A single command or a list of commands to learn.
       example: "Turn on"
+    command_type:
+      description: The type of command to be learned.
+      example: "rf"
     alternative:
       description: If code must be stored as alternative (useful for discrete remotes).
       example: "True"
     timeout:
       description: Timeout, in seconds, for the command to be learned.
       example: "30"
+
+delete_command:
+  description: Deletes a command or a list of commands from the database.
+  fields:
+    entity_id:
+      description: Name(s) of the remote entities holding the database.
+      example: "remote.bedroom"
+    device:
+      description: Name of the device from which commands will be deleted.
+      example: "television"
```

Comment:
Do we need `command_type` here?

Reply:
I guess we already know what kind is was assuming the device name is unique. Is it possible that one device could do IR and RF?

Reply:
Its probably better to not have a second set of storage for the `command_type` so I don't think we need it.